packages/stark-ui/src/modules/input-mask-directives/directives/text-mask-config.intf.ts

```diff
@@ -41,12 +40,20 @@ export interface StarkTextMaskConfig extends StarkTextMaskBaseConfig {
 	 *
 	 * See {@link https://github.com/text-mask/text-mask/blob/master/componentDocumentation.md#mask}
 	 */
-	mask: Mask | false;
+	mask: any;
```

Comment:
Could you provide a type instead of `any` and update the documentation of the property ?